proprietary/tokens/src/common/amsterdam/link-type.tokens.json

```
@@ -0,0 +1,47 @@
+{
+  "amsterdam": {
+    "link-type": {
+      "color": { "value": "{amsterdam.color.primary-blue}" },
+      "hover": {
+        "color": { "value": "{amsterdam.color.dark-blue}" }
+      },
+      "on-background-dark": {
+        "color": { "value": "{amsterdam.color.primary-white}" },
+        "hover": {
+          "color": { "value": "{amsterdam.color.primary-white}" }
+        }
+      },
+      "on-background-light": {
+        "color": { "value": "{amsterdam.color.primary-black}" },
+        "hover": {
+          "color": { "value": "{amsterdam.color.primary-black}" }
+        }
+      },
+      "regular": {
+        "text-decoration-line": { "value": "underline" },
+        "text-decoration-thickness": { "value": "2px" },
+        "text-underline-offset": { "value": "6px" },
+        "hover": {
+          "text-decoration-thickness": { "value": "3px" },
+          "text-underline-offset": { "value": "5px" }
+        }
+      },
+      "apparent": {
+        "text-decoration-line": { "value": "none" },
+        "text-decoration-thickness": { "value": "3px" },
+        "text-underline-offset": { "value": "3px" },
+        "hover": {
+          "text-decoration-line": { "value": "underline" }
+        }
+      },
+      "in-list": {
+        "text-decoration-line": { "value": "none" },
+        "text-decoration-thickness": { "value": "2px" },
+        "text-underline-offset": { "value": "6px" },
+        "hover": {
+          "text-decoration-line": { "value": "underline" }
```

The underline should be 3px on hover here as well.

```suggestion
          "text-decoration-line": { "value": "underline" },
          "text-decoration-thickness": { "value": "3px" }
```

We could even lift that token up a level as it’s the same for all link types?

---

See one of my previous comments, in Figma there's a difference between 'in-list' and 'apparent'. I think we could merge them, I'd opt for 2px thickness everywhere though. 3px is a bit heavy.

Edit: I just checked how it looks with a 6px offset, 3px thickness looks fine then. So I don't really have a preference

---

It’s 2px initial and 3px on hover everywhere, right?